command-parser.js

```diff
@@ -113,6 +113,22 @@ class CommandContext {
 		this.inputUsername = '';
 	}
 
+	checkFormat(room, message) {
+		if (!room) return false;
+		if (!room.stretching && !room.capitals) return false;
+		let formatError = false;
+		let stretchMatch = room.stretching && message.match(/(.+?)\1{7,}/i);
+		let capsMatch = room.capitals && message.match(/[A-Z\s]{18,}/);
```

Comment:
18 and 7 are pretty arbitrary. Do you have the matching statistics to back it up?